requirements.txt

```diff
@@ -6,7 +6,7 @@
 # git tag -a -f cassandra-test
 # git push origin :refs/tags/cassandra-test
 # git push -f origin refs/tags/cassandra-test
--e git+https://github.com/riptano/ccm.git@cassandra-test#egg=ccm
+-e git+https://github.com/ekaterinadimitrova2/ccm.git@16644#egg=ccm
```

Comment:
Return the riptano CCM, I was using my repo only for test purposes.

Reply:
This I already mentioned in the ticket